index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Resume</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Resume">
+    <meta name="author" content="andreCoronado.com">
+    
+    <link href="https://fonts.googleapis.com/css?family=McLaren|Montserrat&display=swap"  rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Darker+Grotesque|Nunito&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,300" />
+    <link rel="stylesheet" href="style.css" />
+    
+</head>
+
+<body onLoad="getResume()">

```

Comment:
I would recommend adding more semantic structure to the body (e.g. header, main, section, footer), but this definitely works for the purpose of the project :)